### PR TITLE
aHrefs SEO Traffic Monitoring Widget Trace Added

### PR DESCRIPTION
**Pull request by dhavalveera**
### New Technology Request

**`Examples:`**

https://www.efireplacestore.com/
https://www.additudemag.com/
https://descuentos.abc.es/
https://podcasters.spotify.com/resources/learn
https://eldiariony.com/

### CODE REVIEW

**Comment by dhavalveera** [comment]
@AliasIO @nurbek91  - can you tell/guide me that it's possible to identify any technology using DNS Records (txt) for Web Extension? if it's possible how to identify the DNS (txt) Records?

File: src/technologies/a.json [hunk 1]
```diff
@@ -4855,6 +4890,22 @@
     "scriptSrc": "/wp-content/plugins/a3-lazy-load/.+\\.js(?:\\?ver=(\\d+(?:\\.\\d+)+))?\\;version:\\1",
     "website": "https://a3rev.com/shop/a3-lazy-load/"
   },
+  "aHrefs": {
+    "cats": [
+      5,
+      10
+    ],
+    "description": "Ahrefs is an All-in-one SEO Toolsets To Grow Your Website Search Traffic & Ahrefs Webmaster Tool to Monitor your SEO Health, know your backlinks, see Keywords that gets Traffic",
```
Comment (nurbek91) [inline src/technologies/a.json:4898]: Ahrefs is an online toolset utilised for search engine optimisation (SEO) and competitor analysis, which permits users to analyse their website's performance, track keyword rankings, identify backlink opportunities, and research competitors' websites, among other features.

File: src/technologies/a.json [hunk 2]
```diff
@@ -4855,6 +4890,22 @@
     "scriptSrc": "/wp-content/plugins/a3-lazy-load/.+\\.js(?:\\?ver=(\\d+(?:\\.\\d+)+))?\\;version:\\1",
     "website": "https://a3rev.com/shop/a3-lazy-load/"
   },
+  "aHrefs": {
+    "cats": [
+      5,
```
Comment (nurbek91) [inline src/technologies/a.json:4895]: lets remove widget category 5 
and replace with 54 SEO

File: src/technologies/a.json [hunk 3]
```diff
@@ -4855,6 +4890,22 @@
     "scriptSrc": "/wp-content/plugins/a3-lazy-load/.+\\.js(?:\\?ver=(\\d+(?:\\.\\d+)+))?\\;version:\\1",
     "website": "https://a3rev.com/shop/a3-lazy-load/"
   },
+  "aHrefs": {
```
Comment (nurbek91) [inline src/technologies/a.json:4893]: aHrefs - > Ahrefs

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR!!

**Review by nurbek91 — APPROVED** [review]

**Pull request merged**